tests/e2e/infra/vsphere.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: holodeck.nvidia.com/v1alpha1
+kind: Environment
+metadata:
+  name: holodeck-vsphere-e2e-test
+  description: "end-to-end test infrastructure"
+spec:
+  provider: vsphere
+  auth:
+    keyName: cnt-ci
+    privateKey: /home/nvidia/.ssh/gpu_runner
+  VsphereVirtualMachine:
+    vCenterURL: "pdx01egxvc01.nvidia.com"
+    datacenter: "PDX01"
+    datastore: "PDX01EGX_vSAN_001"
+    cluster: "PDX01EGX001"
+    network: "PDX01EGXDVS001_Internal_307_10.185.111.0_24"
+    vmFolder: "GitHub ARC"
+    resoursePool: "CNT-ARC"
+    templateImage: "GPU_UBUNTU_TEMPLATE"
```

Comment:
Not critical for this PR, but this seems like the incorrect place to define this. If this is shared across projects, should it not be defined centrally -- or even on the runners themselves -- so that required changes do not need to be made to every project where this is used.

Reply:
Makes sense, this could be handled by Holodeck or the k8s-test-infra repo

Reply:
I'm ok to do that as a follow up.